src/fsharp/FSharp.Core/control.fsi

```diff
@@ -268,6 +268,13 @@ namespace Microsoft.FSharp.Control
         /// <returns>A computation that returns an array of values from the sequence of input computations.</returns>
         static member Parallel : computations:seq<Async<'T>> -> Async<'T[]>
 
+        /// <summary>Creates an asynchronous computation that executes all the given asynchronous computations, 
+        /// and returns the result of the first succeeding computation (i.e. the first computation with a result that is not None).</summary>
```

Comment:
`that is Some` instead of `that is not None` ( i read it like a double negation )